sdk/tests/sdk/whirlpools/quote/increase-liquidity-quote.test.ts

```diff
@@ -0,0 +1,135 @@
+import *  as assert from "assert";
+import { PriceMath, increaseLiquidityQuoteByInputTokenWithParams } from "../../../../src";
+import { BN } from "bn.js";
+import { PublicKey } from "@solana/web3.js";
+import { Percentage } from "@orca-so/common-sdk";
+
+describe("edge cases", () => {
+  const tokenMintA = new PublicKey("orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE");
+  const tokenMintB = new PublicKey("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v");
+
+  it("sqrtPrice on lower bound, tokenB input", async () => {
+    const quote = increaseLiquidityQuoteByInputTokenWithParams({
+      inputTokenAmount: new BN(1000),
+      inputTokenMint: tokenMintB,
+      sqrtPrice: PriceMath.tickIndexToSqrtPriceX64(0),
+      tokenMintA,
+      tokenMintB,
+      tickLowerIndex: 0,
+      tickUpperIndex: 64,
+      tickCurrentIndex: 0,
+      slippageTolerance: Percentage.fromFraction(0, 100),
+    });
+
+    assert.ok(quote.liquidityAmount.isZero());
```

Comment:
For my understanding: basically this means you want to deposit x about of tokenB even though with these prices it should be a single-sided token A deposit. Meaning you can try to add as much of tokenB as you want but since it is single-sided token A deposit you'll always get a zero-liquidity quote.